pom.xml

```diff
@@ -127,6 +127,7 @@
         <module>kafka-init</module>
         <module>certificate-manager</module>
         <module>systemtest</module>
+        <module>kafka-thirdparty-libs</module>
```

Comment:
Do we need it as a submodule when it is only used from the docker build and has nothing to do with the rest of the Maven build? I think we can just remove it. WDYT @tombentley?

Reply:
I agree we don't need it as a submodule with the way the build works currently.

Reply:
Ok , i will remove submodule, but please specify a right place where i will palce it?

Reply:
I don't think it needs to be put anywhere. It will be driven via the `make` build. It doesn't have to be a submodule of anything for that to happen, I think.